.vuepress/public/_redirects

```diff
@@ -1,84 +1,84 @@
-/references/stop_words.html                         /reference/api/stop_words.html
-/references/ranking_rules.html                      /reference/api/ranking_rules.html
-/references/attributes_for_faceting.html            /reference/api/attributes_for_faceting.html
-/references/distinct_attribute.html                 /reference/api/distinct_attribute.html
-/references/searchable_attributes.html              /reference/api/searchable_attributes.html
-/references/displayed_attributes.html               /reference/api/displayed_attributes.html
+/references/synonyms.html                           /reference/api/settings.html
+/references/stop_words.html                         /reference/api/settings.html
+/references/ranking_rules.html                      /reference/api/settings.html
+/references/attributes_for_faceting.html            /reference/api/settings.html
+/references/distinct_attribute.html                 /reference/api/settings.html
+/references/searchable_attributes.html              /reference/api/settings.html
+/references/displayed_attributes.html               /reference/api/settings.html
 /references/stats.html                              /reference/api/stats.html
 /references/health.html                             /reference/api/health.html
 /references/version.html                            /reference/api/version.html
 /references/dumps.html                              /reference/api/dumps.html
-/resources                                          /create/how_to
-/resources/about_storage.html                       /reference/under_the_hood/storage.html
+/resources                                          /learn/cookbooks/running_production.html
+/resources/about_storage.html                       /learn/advanced/storage.html
 /resources/sdks.html                                /learn/what_is_meilisearch/sdks.html
 /resources/comparison_to_alternatives.html          /learn/what_is_meilisearch/comparison_to_alternatives.html
-/resources/postman_collection.html                  /create/how_to/postman_collection.html
+/resources/postman_collection.html                  /learn/cookbooks/postman_collection.html
 /resources/contact.html                             /learn/what_is_meilisearch/contact.html
-/resources/tutorials                                /learn/getting_started
-/running-production                                 /create/how_to/running_production.html
-/resources/howtos                                   /create/how_to
-/instant-search-react-app                           /create/how_to/meilisearch_react.html
-/resources/howtos/digitalocean_droplet.html         /create/how_to/digitalocean_droplet.html
-/resources/howtos/search_bar_for_docs.html          /create/how_to/search_bar_for_docs.html
-/http2-ssl                                          /create/how_to/http2_ssl.html
+/resources/tutorials                                /learn/getting_started/quick_start.html
```

Comment:
Should this redirect to cookbooks?

Reply:
hmmm…I think Quick Start is better. Our current site structure means we always need to send users to a specific page rather than a directory. I would rather someone trying to visit `/tutorials/` ended up in our generic tutorial covering Meilisearch basics than in a super specific article like "Run in production" or "Add a search bar to your docs".